test/em_real/namelist.input.global

```diff
@@ -34,8 +34,8 @@
  p_top_requested                     = 5000,
  num_metgrid_levels                  = 32,
  num_metgrid_soil_levels             = 4,
- dx                                  = 156343.322,20000, 4000,
- dy                                  = 156343.322,20000, 4000,
+ dx                                  = 156343.322,20000,
```

Comment:
@davegill 
Out of curiosity, why does this one still need d02 info?

Reply:
@kkeene44 
Kelly,
Muchas gracias
🦅 👁 👍